README.md

```diff
@@ -47,13 +51,14 @@ Generally speaking, technical decisions should be made by consensus within STAC
 
 * The Chair is the ultimate adjudicator if things break down.
 * The maximum number of PSC members is 7.
+* The maximum number of PSC members per company is 1.
```

Comment:
Not sure whether this needs to be put in writing or whether it should be 1 or 2...